automates/text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndAlign.scala

```diff
@@ -878,46 +914,46 @@ object ExtractAndAlign {
     }
 
     // Equation -> Text
-    if (linkElKeys.contains(EQUATION) && linkElKeys.contains(TEXT_VAR)) {
+    if (linkElKeys.contains(EQUATION) && linkElKeys.contains(GLOBAL_VAR)) {
       println("has eq and text")
-      hypotheses.appendAll(mkLinkHypothesis(linkElements(EQUATION), linkElements(TEXT_VAR), EQN_TO_TEXT, alignments(EQN_TO_TEXT), debug))
+      hypotheses.appendAll(mkLinkHypothesis(linkElements(EQUATION), linkElements(GLOBAL_VAR), EQN_TO_TEXT, alignments(EQN_TO_TEXT), debug))
     }
 
     // Comment -> Text Var
-    if (linkElKeys.contains(COMMENT) && linkElKeys.contains(TEXT_VAR)) {
-      hypotheses.appendAll(mkLinkHypothesis(linkElements(COMMENT), linkElements(TEXT_VAR), COMMENT_TO_TEXT, alignments(COMMENT_TO_TEXT), debug))
+    if (linkElKeys.contains(COMMENT) && linkElKeys.contains(GLOBAL_VAR)) {
+      hypotheses.appendAll(mkLinkHypothesis(linkElements(COMMENT), linkElements(GLOBAL_VAR), COMMENT_TO_TEXT, alignments(COMMENT_TO_TEXT), debug))
     }
 
     // TextVar to Unit (through var)
-    if (linkElKeys.contains(TEXT_VAR) && linkElKeys.contains(UNIT_THRU_VAR)) {
-      hypotheses.appendAll(mkLinkHypothesis(linkElements(TEXT_VAR), linkElements(TEXT_VAR), TEXT_VAR_TO_UNIT, alignments(TEXT_VAR_TO_UNIT), debug))
+    if (linkElKeys.contains(GLOBAL_VAR) && linkElKeys.contains(UNIT_THRU_VAR)) {
+      hypotheses.appendAll(mkLinkHypothesis(linkElements(GLOBAL_VAR), linkElements(GLOBAL_VAR), TEXT_VAR_TO_UNIT, alignments(TEXT_VAR_TO_UNIT), debug))
     }
 
     // TextVar to Unit (through concept)
-    if (linkElKeys.contains(TEXT_VAR) && linkElKeys.contains(UNIT_THRU_CONCEPT)) {
-      hypotheses.appendAll(mkLinkHypothesis(linkElements(TEXT_VAR), linkElements(TEXT_VAR), TEXT_TO_UNIT, alignments(TEXT_TO_UNIT), debug))
+    if (linkElKeys.contains(GLOBAL_VAR) && linkElKeys.contains(UNIT_THRU_CONCEPT)) {
+      hypotheses.appendAll(mkLinkHypothesis(linkElements(GLOBAL_VAR), linkElements(GLOBAL_VAR), TEXT_TO_UNIT, alignments(TEXT_TO_UNIT), debug))
     }
 
 
     // TextVar to ParamSetting (through var)
-    if (linkElKeys.contains(TEXT_VAR) && linkElKeys.contains(PARAM_SETTING_THRU_VAR)) {
-      hypotheses.appendAll(mkLinkHypothesis(linkElements(TEXT_VAR), linkElements(PARAM_SETTING_THRU_VAR), TEXT_VAR_TO_PARAM_SETTING, alignments(TEXT_VAR_TO_PARAM_SETTING), debug))
+    if (linkElKeys.contains(GLOBAL_VAR) && linkElKeys.contains(PARAM_SETTING_THRU_VAR)) {
+      hypotheses.appendAll(mkLinkHypothesis(linkElements(GLOBAL_VAR), linkElements(PARAM_SETTING_THRU_VAR), TEXT_VAR_TO_PARAM_SETTING, alignments(TEXT_VAR_TO_PARAM_SETTING), debug))
     }
 
     // TextVar to ParamSetting (through concept)
     if (linkElKeys.contains(TEXT_VAR) && linkElKeys.contains(PARAM_SETTING_THRU_CONCEPT)) {
```

Comment:
@maxaalexeeva I have a feeling the check here (line 944) was supposed to be `linkElKeys.contains(GLOBAL_VAR)`. Does this need to be fixed?